src/utils/api.js

```diff
@@ -37,4 +37,19 @@ export const updateArticleVotes = (id, voteChange) => {
    });
 };
 
+export const postNewComment = (id, username, comment) => {
+   const data = {
+      username: username,
+      body: comment,
+   };
+
+   return axios.post(`${baseUrl}articles/${id}/comments`, data)
+   .then(response => {
+      return response.data.comment; 
+   })
+   .catch(error => {
+      throw error; 
+   });
+}
```

Comment:
Nice!

Reply:
And yes, to answer your question, you could let the user know the username is invalid by having a validator function when posting the comment and if it fails the validation then you can give the user a specific error message. or the other option is to update the backend to give a specific error message and then display this in the fe